simulator/src/app.ts

```diff
@@ -52,6 +53,9 @@ app.use('/api/house', houseapi);
 // Mount REST api route /api/power-plant to power-plantapi
 app.use('/api/power-plant', powerPlantApi);
 
+// Mount REST api route /api/market to marketapi
+app.use('/api/market', marketApi);
+
```

Comment:
This is incompatible with my branch, either wait until my branch is merged to master and update then.

Reply:
Why should this branch wait? I can't think of a good reason for waiting for the other pull request.